modify-vm-resources/modify-vm-resources.sw.yaml

```diff
@@ -21,6 +21,10 @@ functions:
   - name: increaseVMStatusRunningRetries
     type: expression
     operation: ".vmStatusRunningRetries=.vmStatusRunningRetries + 1"
+  - name: fillEmptyValues
+    type: expression
+    operation: '{vm_new_memory: (if (.inputs.vm_new_memory == null or .inputs.vm_new_memory == "") then .vm.spec.template.spec.domain.memory.guest else .inputs.vm_new_memory end), vm_new_cpu_cores: (if .inputs.vm_new_cpu_cores == null then .vm.spec.template.spec.domain.cpu.cores else .inputs.vm_new_cpu_cores end), vm_new_cpu_sockets: (if .inputs.vm_new_cpu_sockets == null then .vm.spec.template.spec.domain.cpu.sockets else .inputs.vm_new_cpu_sockets end), vm_new_cpu_threads: if .inputs.vm_new_cpu_threads == null then .vm.spec.template.spec.domain.cpu.threads else .inputs.vm_new_cpu_threads end}'
```

Comment:
is there an option to have this section formatted to ease its readability?

Reply:
Done, waiting the demo cluster to be provisionned to test it actually works, otherwise I'll revert the commit

Reply:
Tested and working now